eng/imports/Packages.targets

```diff
@@ -18,49 +18,51 @@
+    <PackageReference Update="Microsoft.VisualStudio.Shell.Design"                                    Version="17.3.0-preview-2-32531-469" />
+    <PackageReference Update="Microsoft.VisualStudio.Shell.Framework"                                 Version="17.3.0-preview-2-32531-469" />
+    <PackageReference Update="Microsoft.VisualStudio.Telemetry"                                       Version="16.5.2" />
+    <PackageReference Update="Microsoft.VisualStudio.TemplateWizardInterface"                         Version="17.3.0-preview-2-32531-469" />
     <PackageReference Update="Microsoft.VisualStudio.Threading"                                       Version="17.3.22-alpha" />
     <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers"                             Version="17.3.22-alpha" />
-    <PackageReference Update="Microsoft.VisualStudio.Utilities"                                       Version="17.1.0-preview-2-32002-062" />
+    <PackageReference Update="Microsoft.VisualStudio.Utilities"                                       Version="17.4.0-preview-3-32908-042" />
     <PackageReference Update="Microsoft.VisualStudio.Validation"                                      Version="17.0.58" />
-    <PackageReference Update="Microsoft.VisualStudio.XmlEditor"                                       Version="17.1.0-preview-2-32002-062" />
+    <PackageReference Update="Microsoft.VisualStudio.XmlEditor"                                       Version="17.3.0-preview-2-32531-469" />
     <PackageReference Update="System.Threading.Tasks.Dataflow"                                        Version="6.0.0" />
     <PackageReference Update="Microsoft.VSDesigner"                                                   Version="17.0.0-preview-2-31223-026" />
     <PackageReference Update="VsWebSite.Interop"                                                      Version="16.8.30523.219" />
 
     <!-- https://github.com/dotnet/roslyn/issues/53877 -->
-    <PackageReference Include="StreamJsonRpc"                                                         Version="2.11.35" />
+    <PackageReference Include="StreamJsonRpc"                                                         Version="2.13.21-alpha" />
 
     <!-- CPS -->
     <!-- Find versions at https://dev.azure.com/azure-public/vside/_artifacts/feed/vs-impl -->
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem"                                   Version="17.4.38-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Analyzers"                         Version="17.4.38-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Query"                             Version="17.4.38-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.SDK.Tools"                         Version="17.4.38-pre" />
-
+    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem"                                   Version="17.5.202-pre-g89e17c9f72" />
```

Comment:
Same comments as above - is the Implementation package reference really needed.
If using SDK, you can remove the reference to MS.VS.PS